src/formatters/Utils.cc

```diff
@@ -55,45 +55,56 @@ namespace {
 /**
 Format a PropertySet into a FITS header string (exactly 80 characters per "card", no line terminator)
 
-This function is designed to format data for creating a WCS. It truncates long string
-values and skips properties whose type it cannot handle.
+This function is designed to format data for creating a WCS. As such, it is quite limited:
```

Comment:
Is there a need to repeat this comment in the "cc" file since it is already in the header?

Reply:
The reason I did it is because it applies to formatFitsProperties in the .h file and formatFitsPropertiesImpl in the .cc file. I'll replace the detailed description with `See @ref formatFitsProperties for details.`